docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+   * The top three lines are pods. These include the actual Trino containers.
+
+   * The next line is the service. This is a mechanism by which Kubernetes makes
+     internal services like the one running on the coordinator accessible
+     within the cluster by using a name instead of IPs.
+
+   * The next two lines are the deployments. Deployments contain logic around
+     how upgrades are performed, and manage pods through replica sets.
+     In the case of ``my-trino-cluster-worker`` deployment, there are multiple
+     pods. Notice that the ``my-trino-cluster-coordinator`` always has one pod.
+
+   * The final two lines are the replica sets. This defines how many instances
+     of a given pod should be available during a ready state. This is the way
+     to scale out the deployments.
+
+#. If all pods, deployments, and replica sets are running and in the ready
+   state, Trino has been successfully deployed.
+
+.. note::
+
+      Unlike some Kubernetes applications, where it's better to have many small
+      pods, Trino works best with fewer pods each having more resources
+      available. We strongly recommend to avoid having multiple Trino pods on a
+      single physical host to avoid contention for resources.
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you need to create a tunnel to the coordinator pod and your computer. You can do
+this by running the commands generated upon installation.
+
+#. Store the coordinator pod name in a shell variable called ``POD_NAME``.
+
+   .. code-block:: text
+
+       POD_NAME=$(kubectl get pods -l "app=trino,release=my-trino-cluster,component=coordinator" -o name)
+
+#. Create the tunnel from the coordinator pod to the client.
+
+   .. code-block:: text
+
+       kubectl port-forward $POD_NAME 8080:8080
+
+   Now you can connect to the Trino coordinator at ``http://localhost:8080``.
+
+#. To connect to Trino, you can use the
+   :doc:`commond-line interface </client/cli>`, a
+   :doc:`jdbc client </client/jdbc>`, or any of the
+   :doc:`other clients </client>`. For this example,
+   :ref:`install the command-line interface <cli-installation>`, and connect to
+   Trino in a new console session.
+
+   .. code-block:: text
+
+       trino --server http://localhost:8080
+
+#. Using the sample data in the ``tpch`` catalog, type and execute a query on
+   the ``nation`` table using the ``tiny`` schema. Try other
+   commands to explore this data set:
```

Comment:
schema:

code block 

Try other SQL queries to explore the data set and test your cluster.